server/openapi_environment.go

```diff
@@ -33,7 +33,7 @@ func (s *Server) listEnvironment(c echo.Context) error {
 		return echo.NewHTTPError(http.StatusInternalServerError, "Failed to fetch environment list").SetInternal(err)
 	}
 
-	var response []*openAPIV1.Environment
+	response := []*openAPIV1.Environment{}
```

Comment:
`var response []*openAPIV1.Environment` will returns `nil` if the `envList` is empty